templates/steering-display/src/main.cpp

```diff
@@ -0,0 +1,51 @@
+/*
+@file    main.cpp
+@brief   Main file for Steering Display Firmware
+@version 1.0
+@date    
+@author  
+*/
+
+#include <Arduino.h>
+#include <SPI.h>
+
+#include "EVE_target.h"
+#include "EVE_commands.h"
+#include "tft.h"
+
+void setup() {
+    Serial2.begin(115200);
+    pinMode(EVE_CS, OUTPUT);
+    digitalWrite(EVE_CS, HIGH);
+    pinMode(EVE_PDN, OUTPUT);
+    digitalWrite(EVE_PDN, LOW);
+
+    SPI.begin(); // Set up the SPI to run in Mode 0 and 8 MHz
+    SPI.beginTransaction(SPISettings(8000000, MSBFIRST, SPI_MODE0));
```

Comment:
Start at 1 MHz? We will check how it behaves we run it with the engine in the dyno. I am concerned about the added noise and whether we will achieve 8 MHz